templates/taxi/car_detail.html

```diff
@@ -12,7 +12,20 @@ <h1>
     </a>
   </h1>
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
+  <h1>
+    Drivers
+
+    {% if car in user.cars.all %}
+      <a href="{% url 'taxi:toggle-car-assign' pk=car.id %}" class="btn btn-danger link-to-page">
+        Delete me from this car
+      </a>
+    {% else %}
+      <a href="{% url 'taxi:toggle-car-assign' pk=car.id %}" class="btn btn-success link-to-page">
+        Assign me from this car
```

Comment:
The button text 'Assign me from this car' is misleading. It should be 'Assign me to this car' to accurately describe the action of adding the user to the car's drivers.